NASA-AMMOS/aerie-ui
# Add permissions to plan creation page

~~This is blocked by #679~~ (Merged)

Resolves #686 

Testing this involves changing code locally, unfortunately. There is currently no way to simulate a user without certain permissions.

To test:
1. Set the value of `canCreate` in `routes/plans/+page.svelte` to false
2. On the plan creation page, verify that the fields needed to create a plan are not interactive and display the appropriate permissions message
